Q42/vue-dotnet-validator
# Feature/update setup

### Changes
Updates the build, example & file packaging setup. To fix bugs, and allow modern bundlers to tree shake this module.

### Testing
- Try this module in an external project with npm link or install `vue-dotnet-validator@0.4.0-beta.1`
- Check that only the parts of this module that you actually use are bundled
